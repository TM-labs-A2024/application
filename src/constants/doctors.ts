export const doctors = [
  {
    id: 1000000,
    specialities: [1],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 2000000,
    specialities: [2, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 3000000,
    specialities: [3, 5, 7],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 4000000,
    specialities: [4, 1],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 5000000,
    specialities: [5, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 6000000,
    specialities: [6, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 7000000,
    specialities: [7, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 8000000,
    specialities: [8, 9, 5, 1, 3, 4],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 9000000,
    specialities: [9, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 10000000,
    specialities: [10, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 11000000,
    specialities: [11, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 12000000,
    specialities: [12, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 13000000,
    specialities: [13, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 14000000,
    specialities: [14, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 15000000,
    specialities: [15, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 16000000,
    specialities: [16, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 17000000,
    specialities: [17, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 18000000,
    specialities: [18, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 19000000,
    specialities: [19, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 20000000,
    specialities: [20, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 21000000,
    specialities: [21, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 22000000,
    specialities: [22, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522',
    pending: true
  },
  {
    id: 23000000,
    specialities: [23, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 24000000,
    specialities: [24, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 25000000,
    specialities: [25, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 26000000,
    specialities: [26, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 27000000,
    specialities: [27, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 28000000,
    specialities: [28, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 29000000,
    specialities: [29, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 30000000,
    specialities: [30, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 31000000,
    specialities: [31, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 32000000,
    specialities: [32, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 33000000,
    specialities: [33, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 34000000,
    specialities: [34, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 35000000,
    specialities: [35, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 36000000,
    specialities: [36, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 37000000,
    specialities: [37, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 38000000,
    specialities: [38, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 39000000,
    specialities: [39, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 40000000,
    specialities: [40, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 41000000,
    specialities: [41, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 42000000,
    specialities: [42, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 43000000,
    specialities: [43, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 44000000,
    specialities: [44, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 45000000,
    specialities: [45, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 46000000,
    specialities: [46, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 47000000,
    specialities: [47, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  },
  {
    id: 48000000,
    specialities: [48, 9, 5],
    firstname: 'José',
    lastname: 'Pérez',
    birthdate: 'Wed Sep 17 1997 00:00:00 GMT-0400 (hora de Venezuela)',
    email: 'joseperez@gmail.com ',
    phone: '4127775522'
  }
]
