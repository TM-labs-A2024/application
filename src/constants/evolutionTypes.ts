export const evolutionTypes = [
  {
    id: '1',
    name: 'Evolución'
  },
  {
    id: '2',
    name: 'Hospitalización'
  },
  {
    id: '3',
    name: 'Alta'
  },
  {
    id: '4',
    name: 'Primera consulta'
  }
]
