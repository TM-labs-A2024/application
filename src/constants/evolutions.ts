export const evolution = {
  specialty: {
    description: '',
    id: '',
    name: ''
  },
  author: '',
  title: '',
  description: '',
  'content-format': '',
  created_at: '',
  payload: {
    type: {
      value: '',
      label: ''
    },
    reason: '',
    description: '',
    history: '',
    examination: '',
    summary: '',
    diagnostic: '',
    comments: ''
  }
}
