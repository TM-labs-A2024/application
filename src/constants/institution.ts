export const institution = {
  id: '3fa85f64-5717-4562-b3fc-2c963f66afa1',
  name: 'Clínica Tex Mex',
  credentials: '14578245',
  address: 'Av. Fernandez Peña, 147.',
  email: 'test@test.com',
  phone: '+582742214441'
}
