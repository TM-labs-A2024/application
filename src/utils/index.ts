import { formatDate } from './dates'
import { sendEmail, generateVerificationCode, setupEmailSending } from './email'
import { setupErrorNotification } from './notifications'
import { isWeb, isIOS, isAndroid, isMobile } from './platform'

export {
  isWeb,
  isIOS,
  isAndroid,
  isMobile,
  sendEmail,
  generateVerificationCode,
  setupEmailSending,
  setupErrorNotification,
  formatDate
}
