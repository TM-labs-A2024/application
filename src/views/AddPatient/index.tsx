import AddPatient from '@components/organisms/AddPatient'
import { PatientsLayout } from '@layouts/index'
import React from 'react'

export default function AddPatientView() {
  return (
    <PatientsLayout>
      <AddPatient />
    </PatientsLayout>
  )
}
