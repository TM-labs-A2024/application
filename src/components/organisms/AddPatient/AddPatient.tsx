import { Heading, List, Text, Button } from '@chakra-ui/react'
import SearchInputComponent from '@components/atoms/SearchInput'
import { patients } from '@src/constants'
import { isIOS } from '@utils/index'
import React from 'react'

export default function AddPatient() {
  // --- Hooks -----------------------------------------------------------------
  // --- END: Hooks ------------------------------------------------------------

  // --- Local state -----------------------------------------------------------
  // --- END: Local state ------------------------------------------------------

  // --- Data and handlers -----------------------------------------------------
  // --- END: Data and handlers ------------------------------------------------

  return (
    <div
      className={`mx-auto block h-screen w-screen overflow-hidden px-8 lg:px-96 ${isIOS() ? 'pt-20' : 'pt-8'}`}
    >
      <Heading as="h2" size="md" className="mb-4">
        Nuevo paciente
      </Heading>
      <SearchInputComponent placeholder="Ingresar cÃ©dula" className="mb-8 w-full" />
      <List spacing={3} className="h-full overflow-scroll pb-56">
        {patients.map((patient, idx) => (
          <div
            className="flex flex-row items-center justify-between"
            key={`new-patient-${idx + 1}`}
          >
            <div className="flex flex-col">
              <Text>
                {patient.firstname} {patient.lastname}
              </Text>
              <Text>{patient.govId}</Text>
            </div>
            <Button>Solicitar acceso</Button>
          </div>
        ))}
      </List>
    </div>
  )
}
