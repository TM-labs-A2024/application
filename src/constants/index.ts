import { doctors } from './doctors'
import { evolutionTypes } from './evolutionTypes'
import { institution } from './institution'
import { institutions } from './institutions'
import {
  ACCESS_REMOVED,
  FILTERS_APPLIED,
  ACCESS_REQUESTED,
  ATTACHMENT_DELETED,
  IMAGE_DELETED,
  ACCESS_DENIED,
  ACCESS_GRANTED,
  ACCESS_REMOVAL,
  GENERIC_NOTIFICATION,
  GENERIC_ERROR
} from './notifications'
import { nurse } from './nurses'
import { patient, sexOptions, statusOptions } from './patient'
import { patients } from './patients'
import { specialties, specialities } from './specialties'
import { specialtyData } from './specialtyData'

export {
  specialties,
  specialities,
  doctors,
  nurse,
  patient,
  sexOptions,
  statusOptions,
  specialtyData,
  evolutionTypes,
  patients,
  institution,
  institutions,
  ACCESS_REMOVED,
  FILTERS_APPLIED,
  ACCESS_REQUESTED,
  ATTACHMENT_DELETED,
  IMAGE_DELETED,
  ACCESS_DENIED,
  ACCESS_GRANTED,
  ACCESS_REMOVAL,
  GENERIC_NOTIFICATION,
  GENERIC_ERROR
}
