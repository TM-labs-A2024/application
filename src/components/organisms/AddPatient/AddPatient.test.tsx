import { patients } from '@src/constants'
import { render, screen } from '@test/utils/index'

import '@testing-library/jest-dom'
import AddPatient from './AddPatient'

const context = {
  onChange: (id: string) => id,
  filteredPatients: patients,
  onSubmit: jest.fn(),
  loadingId: ''
}

const pendingPatients = {
  onChange: (id: string) => id,
  filteredPatients: [patients.find((patient) => patient.pending) ?? patients[0]],
  onSubmit: jest.fn(),
  loadingId: ''
}

const approvedPatients = {
  onChange: (id: string) => id,
  filteredPatients: [patients.find((patient) => !patient.pending) ?? patients[0]],
  onSubmit: jest.fn(),
  loadingId: ''
}

describe('Organisms > AddPatient test', () => {
  test('The component shows the patients that match the id in the input field', () => {
    render(<AddPatient context={context} />)

    patients.forEach((patient) => {
      expect(screen.getByText(patient.govId)).toBeInTheDocument()
    })
  })

  test('The component shows if the access was already requested', () => {
    render(<AddPatient context={pendingPatients} />)

    patients.forEach((patient) => {
      if (patient.pending) expect(screen.getByText('Solicitado')).toBeInTheDocument()
    })
  })

  test('The component allows to request access to pacients', () => {
    render(<AddPatient context={approvedPatients} />)

    patients.forEach((patient) => {
      if (!patient.pending) expect(screen.getByText('Solicitar acceso')).toBeInTheDocument()
    })
  })

  test('Matches the snapshot', () => {
    render(<AddPatient context={context} />)

    expect(screen.getByTestId('add-patient')).toMatchSnapshot()
  })
})
