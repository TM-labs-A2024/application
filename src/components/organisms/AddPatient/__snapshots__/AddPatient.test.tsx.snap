// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Organisms > AddPatient test Matches the snapshot 1`] = `
<div
  class="mx-auto block h-screen w-screen overflow-hidden px-8 lg:px-96 pt-8"
  data-testid="add-patient"
>
  <h2
    class="chakra-heading mb-4 css-18j379d"
  >
    Nuevo paciente
  </h2>
  <div
    class="mb-8 w-full"
    data-testid="search-input-component"
  >
    <div
      class="chakra-input__group css-1y0e7gb"
      data-group="true"
    >
      <div
        class="chakra-input__left-element css-gv1iwz"
      >
        <svg
          class="chakra-icon css-13izzit"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M23.384,21.619,16.855,15.09a9.284,9.284,0,1,0-1.768,1.768l6.529,6.529a1.266,1.266,0,0,0,1.768,0A1.251,1.251,0,0,0,23.384,21.619ZM2.75,9.5a6.75,6.75,0,1,1,6.75,6.75A6.758,6.758,0,0,1,2.75,9.5Z"
            fill="currentColor"
          />
        </svg>
      </div>
      <input
        class="chakra-input min-h-10 css-qluo93"
        placeholder="Ingresar cédula"
      />
    </div>
  </div>
  <h2
    class="chakra-heading mb-4 css-o8iskg"
  >
    13 resultados
  </h2>
  <ul
    class="h-full overflow-scroll pb-56 css-1xa84ef"
    role="list"
  >
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          José Feliciano
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885561
        </p>
      </div>
      <button
        class="chakra-button cursor-not-allowed css-ez23ye"
        type="button"
      >
        Solicitado
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          José Feliciano
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885562
        </p>
      </div>
      <button
        class="chakra-button cursor-not-allowed css-ez23ye"
        type="button"
      >
        Solicitado
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          Josefa Feliciana
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885563
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          José Feliciano
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885564
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          Josefa Feliciana
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885565
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          José Feliciano
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885566
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          José Feliciano
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885567
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          Josefa Feliciana
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885568
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          José Feliciano
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885569
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          Josefa Feliciana
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885560
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          José Feliciano
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885511
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          José Feliciano
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885512
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
    <div
      class="flex flex-row items-center justify-between"
    >
      <div
        class="flex flex-col"
      >
        <p
          class="chakra-text css-0"
        >
          José Feliciano
           
          Gutierrez García
        </p>
        <p
          class="chakra-text css-0"
        >
          E99885513
        </p>
      </div>
      <button
        class="chakra-button css-ez23ye"
        type="button"
      >
        Solicitar acceso
      </button>
    </div>
  </ul>
</div>
`
